lib/pure/unittest.nim

```diff
@@ -116,7 +117,7 @@ template fail* =
   for msg in items(checkpoints):
     # this used to be 'echo' which now breaks due to a bug. XXX will revisit
     # this issue later.
-    stdout.writeln msg
+    echo msg
```

Comment:
Please pay attention to this line. Is the mentioned bug fixed already? Should I just remove the comment?